test/upgrade/prober/configuration.go

```diff
@@ -62,22 +64,47 @@ func (p *prober) annotateNamespace() {
 	ensure.NoError(err)
 }
 
+func (p *prober) fetchBrokerUrl() (*apis.URL, error) {
+	namespace := p.config.Namespace
+	p.log.Debugf("Fetching %s broker URL for ns %s", brokerName, namespace)
+	meta := resources.NewMetaResource(brokerName, p.config.Namespace, lib.BrokerTypeMeta)
+	err := duck.WaitForResourceReady(p.client.Dynamic, meta)
+	if err != nil {
+		return nil, err
+	}
+	broker, err := p.client.Eventing.EventingV1alpha1().Brokers(namespace).Get(brokerName, metav1.GetOptions{})
```

Comment:
can you follow-up to fetch the v1beta1 version here

Reply:
Sure.

Reply:
But, there are more places where alpha1 is used. Should I change all of them to beta?

Reply:
The plan is to drop the v1alpha1 in .16, so at least before then we should get rid of all the v1alpha1 ones :)

Reply:
Cool. I'll create issue for that, and PR.